aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/control

```diff
@@ -0,0 +1,12 @@
+Source: camera-streaming-daemon
+Priority: optional
+Maintainer: Ritul Jasuja <ritul.jasuja@intel.com>
+Build-Depends: debhelper (>=9), dh-systemd (>= 1.5), dh-autoreconf, git, python-future, libavahi-client-dev, libavahi-core-dev, libavahi-glib-dev, libqt5gstreamer-1.0-0, libgstreamer-plugins-base1.0-dev, libgstrtspserver-1.0-dev
+Standards-Version: 3.9.7
+Homepage: https://github.com/01org/camera-streaming-daemon
+
+Package: aero-camera-streaming-daemon
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
```

Comment:
I bet the way it is now the generated packages won't tell apt about its runtime dependencies; which means csd gets installed without the required libraries.

Reply:
ping

Reply:
I think @Rjasuja for now verified build time dependencies on docker image, we will run the generated debian package on docker image to know the runtime dependencies.

Reply:
Runtime Dependencies are taken care by ${shlibs:Depends} variable. Verified it on fresh ubuntu docker image